### PR TITLE
[test/plugin] spring scenario update

### PR DESCRIPTION
**Pull request by arugal**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: test/plugin/scenarios/spring-3.0.x-scenario/configuration.yml [hunk 1]
```diff
@@ -18,4 +18,5 @@ type: tomcat
 entryService: http://localhost:8080/spring-3.0.x-scenario/case/spring3
 healthCheck: http://localhost:8080/spring-3.0.x-scenario/healthCheck
 runningMode: with_optional
+with_plugins: apm-spring-annotation-plugin-*.jar
```
Comment (kezhenxu94) [inline test/plugin/scenarios/spring-3.0.x-scenario/configuration.yml]: I think it's `withPlugins`

Comment (kezhenxu94) [inline test/plugin/scenarios/spring-3.0.x-scenario/configuration.yml]: I think it's `withPlugins`

Comment (kezhenxu94) [reply test/plugin/scenarios/spring-3.0.x-scenario/configuration.yml]: Same as other places

Comment (arugal) [reply test/plugin/scenarios/spring-3.0.x-scenario/configuration.yml]: done

**Pull request merged**